<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InUpDialog</class>
 <widget class="QDialog" name="InUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设备购进和信息修改</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 0, 127);</string>
  </property>
  <widget class="QPushButton" name="Badd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>195</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>设备ID：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="sbId">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>设备名称：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="sbName">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>设备型号：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="sbType">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>设备产地：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="sbChandi">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>设备数量：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="sbCount">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>供应商：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="sbProvider">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>设备价格：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="sbPrice">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Bup">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="sbDate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>购进日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>使用年限至</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="sbTimeout">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
